docs/src/api.md

```diff
@@ -0,0 +1,28 @@
+# API
+
+In addition to its own functionality, MixedModels.jl also implements extensive support for the `StatsBase.StatisticalModel` and `StatsBase.RegressionModel` API.
+
+## Types
+
+```@autodocs
+Modules = [MixedModels]
+Order   = [:type]
+```
+
+## Exported Functions
+```@autodocs
+Modules = [MixedModels]
+Private = false
+Order   = [:function]
+```
+
+## Non-Exported Functions
+
+Note that unless discussed elsewhere in the online documentation, non-exported functions should be considered implementation details.
+
+```@autodocs
+Modules = [MixedModels]
+Public  = false
+Order   = [:function]
+Filter = f -> !startswith(string(f), "_")
+```
```

Comment:
How is Public/Private determined?  Several of the functions documented in the `Non-Exported` section are in fact exported.

Reply:
My understanding was that it's based on exporting. I'll see what I can dig up...

Reply:
I was also a bit surprised to see that in one case the key is `Private` and in the other it is `Public`.  I suppose I could just read the documentation for myself to see what is going on.

Reply:
I wonder if the keys do different things and it makes sense to specify both...